src/api_manager/method_impl.cc

```diff
@@ -108,6 +108,27 @@ const std::string &MethodInfoImpl::first_authorization_url() const {
   return empty;
 }
 
+void MethodInfoImpl::process_backend_rule(
+    const ::google::api::BackendRule &rule) {
+  backend_address_ = rule.address();
+  backend_path_translation_ = rule.path_translation();
+  backend_jwt_audience_ = rule.jwt_audience();
+  if (backend_path_translation_ ==
+      ::google::api::BackendRule_PathTranslation_CONSTANT_ADDRESS) {
+    // for CONSTANT ADDRESS case, need to split the rule.address into
+    // address and path. Example: "https://example.cloudfunctions.net/getUser"
+    // should be split to "https://example.cloudfunctions.net/getUser" and
```

Comment:
How about case without "http://" ?

Reply:
Same as https.  the logic here is to find the position of third "/", then split it into two.

Reply:
I mean can we handle if 

backend = host/path 

without https://

Reply:
No.  That is not allowed by Inception.   

Inception tool checks the protocol from this backend address. And actually currently they plan to only support https.

Reply:
Could you add the comment for this?